packages/kolme/src/core/types.rs

```diff
@@ -1694,23 +1718,35 @@ impl ExecAction {
                 ChainName::PassThrough => todo!(),
             },
             ExecAction::MigrateContract { migrate_contract } => {
+                let contract_addr = match &config.bridge {
+                    BridgeContract::Deployed(addr) => addr.clone(),
+                    _ => anyhow::bail!(
+                        "Unable to migrate contract for chain {chain}: contract isn't deployed"
+                    ),
+                };
+
                 match chain.name() {
                     #[cfg(feature = "cosmwasm")]
                     ChainName::Cosmos => {
-                        let contract_addr = match &config.bridge {
-                        BridgeContract::Deployed(addr) => addr.clone(),
-                        _ => anyhow::bail!("Unable to migrate contract for chain {chain}: contract isn't deployed")
-                    };
-                        let MigrateContract {
-                            chain: _,
+                        #[allow(irrefutable_let_patterns)]
```

Comment:
This doesn't seem necessary.

Reply:
It looks like there's another warning for the only-cosmwasm case:

```
➜ cargo check --features cosmwasm
    Checking kolme v0.1.0 (/home/michael/fpco/kolme/packages/kolme)
warning: unused import: `absurd_future::absurd_future`
 --> packages/kolme/src/listener/mod.rs:7:5
  |
7 | use absurd_future::absurd_future;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
```

As for the irrefutable let pattern: can we change the code to pattern match on `migrate_contract.inner()` instead of `chain.name()`?